lighthouse-core/test/results/sample_v2.json

```diff
@@ -1719,90 +1719,24 @@
       "title": "Largest Contentful Paint element",
       "description": "This is the largest contentful element painted within the viewport. [Learn More](https://web.dev/lighthouse-largest-contentful-paint/)",
       "score": null,
-      "scoreDisplayMode": "informative",
-      "displayValue": "1 element found",
+      "scoreDisplayMode": "notApplicable",
+      "displayValue": "0 elements found",
```

Comment:
I think you'll need to manually update the artifacts with the new traceEventType approach